src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -12,6 +12,8 @@
 import org.joda.time.DateTimeZone;
 
 import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.logic.api.Logic;
```

Comment:
This is not allowed. `FieldValidator` is a general utility method and should not have a dependency on the business logic.